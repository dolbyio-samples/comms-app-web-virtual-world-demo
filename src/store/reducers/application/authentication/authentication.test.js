import { expect, it } from "vitest";
import reducer from "./authentication";

it("dispatch setToken action, reducer should save token", () => {
  // arrange
  const token = "token";
  const action = { type: "SET_TOKEN", payload: { token } };
  const state = { token: "" };

  // act
  const updatedState = reducer(state, action);

  //assert
  expect(updatedState).toMatchSnapshot();
});
