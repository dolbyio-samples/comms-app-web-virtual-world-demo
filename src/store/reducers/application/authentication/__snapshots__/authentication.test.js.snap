// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`dispatch setToken action, reducer should save token 1`] = `
{
  "token": "token",
}
`;
